iothub/device/tests/Pipeline/ErrorDelegatingHandlerTests.cs

```diff
@@ -103,7 +103,7 @@ public async Task ErrorHandler_TransientErrorOccuredChannelIsAlive_ChannelIsTheS
         [TestMethod]
         public async Task ErrorHandler_SecurityErrorOccured_ChannelIsAborted()
         {
-            await TestExceptionThrown(typeof(TestSecurityException), typeof(AuthenticationException)).ConfigureAwait(false);
+            await TestExceptionThrown(typeof(TestSecurityException), typeof(IotHubClientException)).ConfigureAwait(false);
```

Comment:
Can we add an error code check to this exception assertion? That'll make the assertion specific to this test case.